assets/js/components/AdminMenuTooltip/useShowTooltip.js

```diff
@@ -60,15 +60,16 @@ export function useShowTooltip( tooltipStateKey ) {
 			};
 		}
 
-		setValue( tooltipStateKey, {
+		setValue( 'admin-menu-tooltip', {
 			isTooltipVisible: true,
 			rehideAdminMenu: ! isAdminMenuOpen,
 			rehideAdminSubMenu: isAdminSubMenuHidden,
+			...tooltipSettings,
```

Comment:
What do all of these props look like? If they aren't all serialisable I'm not sure why we're choosing to store them in the datastore rather than pass them as props. 🤔

Or at least just store them locally in state rather than in the datastore.

Reply:
I guess the aim of storing these in the datastore is just to have them centralised so that multiple tooltips for an area aren't appearing. 🤔